security-and-compliance/harden-tap.hbs.md

```diff
@@ -227,6 +231,8 @@ Please refer to [TAP Architecture Overview](https://docs.vmware.com/en/VMware-Ta
 
 ## Logging
 
+Log files provide an audit trail necessary to monitor activity within infrastructure, identify policy violations, unusual activity, and highlight security incidents. It is vital that logs are captured and retained according to the policies set forth by the organization's security team or governing body. TAP components run as pods on the Kubernetes infrastructure and all components output to standard out, captured as part of the pod logs.
```

Comment:
what does '.. all components output to standard out, captured as part of the pod logs.'  mean?